src/utils/convert.js

```diff
@@ -90,17 +82,24 @@ const toSlateMarks = (attributes) => {
  * toSlateDoc(syncDoc: SyncDoc): Node[]
  */
 const toSlateDoc = (syncDoc) => {
-  return syncDoc.map(toSlateNode);
+  const documentNodes = syncDoc.get("document") || [];
+  const nodes = documentNodes.map(toSlateNode);
+  const document = Document.create({ nodes });
+  const data = syncDoc.get("data");
+  return Value.create({ document, data: data ? data.toJSON() : {} });
 };
```

Comment:
I guess there weren't any unit tests covering this code path?

Reply:
Nope, I added a basic one.